src/app/checkout/step-1/step-1.component.js

```diff
@@ -67,6 +69,6 @@ export default angular
     controller: Step1Controller,
     templateUrl: template.name,
     bindings: {
-      changeStep: '&'
```

Comment:
I gave that a try this morning, but I couldn't figure it out.  I think there must be something missing.